### DISCUSSION TITLE
Disabling World Buffs (among other things) - Darrowshire as a Reference

### DISCUSSION THREAD

**Discussion by Lanceroyy** [chromiecraft/chromiecraft#2482] [General]
Hello, 

So I noticed Darrowshire (and now Blizzard is copying Darrowshire) are doing things like disabling world buffs for raids and having no Honor Decay for PVP, but adjusting it in a way that it takes a similar amount of time to reach Rank 14. 
Basically making Vanilla a more enjoyable experience for people in general and rooting out the inconvenient aspects that no body enjoyed in the original game. This seems to be a great model for this content! 

I was just wondering if Chromiecraft will take a similar approach especially when it comes to level 60 content?
 I think things like this would be really beneficial for Chromiecraft to consider!

Thanks
-Lanceroyy

**Comment by 55Honey** [comment]
Hi Lanceroy,

You want to check the news on our website https://www.chromiecraft.com/ and our discord at https://discord.gg/jUdJRhmT3J where most of that was answered already.
